api.yaml

```diff
@@ -14,287 +14,97 @@ externalDocs:
-          in: query
-          description: date specifying to only return events that occured before (inclusive) that date.
-          required: false
-          type: string
-          format: date-time
-          allowEmptyValue: false
-        - name: eventnames
-          in: query
-          description: coma separated list of event names so that only events excatly matching one of theese names is returned
-          required: false
-          type: string
-          format: string
-          allowEmptyValue: false
-        - name: sort
-          in: query
-          description: specifying the sorting on event names or change date when an event occured
-          required: false
-          type: string
-          format: string
-          allowEmptyValue: false
+        - $ref: '#/parameters/StartDate'
+        - $ref: '#/parameters/EndDate'
       responses:
         200:
-          description: Returns an object with application and version ID and a an array of requested data
+          description: Returns an array of objects per version of the requested application with amount running instances in a certain point of time
           schema:
-            type: object
-            properties:
-              application_id:
-                type: string
-                description: Unique identifier of the application
-                example: kio
-              versions:
-                type: array
-                items:
-                   type: object
-                   properties:
-                     version_id:
-                       type: string
-                       description: Unique identifier of the version
-                       example: somestring
-                     data:
-                       type: array
-                       items:
-                         type: object
-                         properties:
-                           change_date:
-                           type: string
-                           format: date-time
-                           description: Point in time when the instance count of the version changed.
-                           example: '2015-04-25T16:25:00.000Z'
-                         event_name:
-                           type: string
-                           description: name of the occured event
-                           example: 'someEventThatHappened'
+            type: array
+            items:
+              type: object
+              properties:
+                version_id:
+                  type: string
+                  description: some version id
+                metadata:
+                  type: object
+                  description: some additional metadata
+                events:
+                  type: array
+                  items:
+                    type: object
+                    properties:
+                      timestamp:
+                        $ref: '#/definitions/Timestamp'
+                      count:
```

Comment:
has been clarified: `count` is the total number of instances, `instances` does only contain affected instances (changes) at this point in time.